### PR TITLE
(ASC-691) refactored create snapshot test to fix the ci failure

### PR DESCRIPTION
**Pull request by phongdly**

### CODE REVIEW

**Comment by phongdly** [comment]
Verified it passed on phobos mnaio:

--> Executing Testinfra tests found in /root/github/rpc-openstack-system-tests/molecules/molecule-validate-nova-deploy/molecule/default/tests/...
    ============================= test session starts ==============================
    platform linux2 -- Python 2.7.12, pytest-3.6.2, py-1.5.4, pluggy-0.6.0
    rootdir: /root/github/rpc-openstack-system-tests/molecules/molecule-validate-nova-deploy/molecule/default, inifile:
    plugins: testinfra-1.12.0, rpc-0.7.0, ordering-0.5
collected 6 items

    tests/test_create_and_assign_floating_ip.pp ...                           [ 50%]
    tests/test_create_snapshot_on_instance.py ...                            [100%]

     generated xml file: /root/github/rpc-openstack-system-tests/molecules/molecule-validate-nova-deploy/molecule/default/molecule_test_results.xml
    ========================== 6 passed in 345.00 seconds ==========================
Verifier completed successfully.

File: molecule/default/tests/test_create_snapshot_on_instance.py [hunk 1]
```diff
@@ -6,61 +6,77 @@
...
-
-    # Create instance
-    image_name = 'Cirros-0.3.5'
-    flavor = 'm1.tiny'
-    network = 'PRIVATE_NET'
-    instance_name = 'test_instance_01'
-    new_instance_name = 'test_instance_02'
-    snapshot_name = 'test_snapshot_02'
+@pytest.mark.run(order=4)
+def test_create_snapshot_on_an_instance(host):
```
Comment (johnduarte) [inline molecule/default/tests/test_create_snapshot_on_instance.py]: nit pick: I think this should be `test_create_snapshot_of_an_instance`.

File: molecule/default/tests/test_create_snapshot_on_instance.py [hunk 2]
```diff
@@ -6,61 +6,77 @@
...
 
     # Verify the snapshot is successfully created:
-    utils.get_expected_status('image', snapshot_name, 'active', host)
+    assert (utils.get_expected_value('image', snapshot_name, 'status', 'active', host))
+
+
+@pytest.mark.test_id('4ca28c34-7e24-11e8-a634-9cdc71d6c128')
+@pytest.mark.jira('asc-691')
+@pytest.mark.run(order=5)
+def test_create_instance_on_snapshot(host):
```
Comment (johnduarte) [inline molecule/default/tests/test_create_snapshot_on_instance.py]: nit pick: I think this should be `test_create_instance_from_snapshot`

**Review by johnduarte — CHANGES_REQUESTED** [review]

**Comment by phongdly** [comment]
@johnduarte : updated!

**Review by johnduarte — APPROVED** [review]

**Pull request merged**